app/companies/controller.js

```diff
@@ -0,0 +1,73 @@
+const db = require('../../db/db');
+const moment = require('moment');
+const { validationResult } = require('express-validator');
+
+module.exports = {
+  getListCompanies: async (req, res) => {
+    try {
+      const perPage = req.query.per_page || 20;
+      const page = req.query.page || 1;
+      const result = await PSQL.query(
+        `SELECT * FROM companies LIMIT ${perPage} OFFSET ${(page - 1) * perPage}`
+      );
+
+      if (!result) {
+        return res.send('No have data!')
+      }
+
+      return res.json(result.rows);
+    } catch (e) {
+      console.log(e.message)
+    }
+  },
+  createCompany: async (req, res) => {
+    try {
+      const errors = validationResult(req);
+      if (!errors.isEmpty()) {
```

Comment:
Duplicated method here, we can create a method for this check to reduce the boilerplate code

Reply:
create new function will be have the same lines here